tests/test_ntp_server.rb

```diff
@@ -83,4 +92,54 @@ def test_multiple
     refute_includes(Cisco::NtpServer.ntpservers, id1)
     refute_includes(Cisco::NtpServer.ntpservers, id2)
   end
+
+  def test_domain_name
```

Comment:
Does `test_domain_name` pass for you?  When I run it on my testbed the configuration is accepted but the ntp server does not actually get configured...  investigating to see if it's just the version of software I am running.